### PR TITLE
TIP-863: Move Values to Enrichment 

### PR DESCRIPTION
**Pull request by jjanvier**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | Y
| Review and 2 GTM                  | -
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: src/Akeneo/Pim/Enrichment/Component/spec/Product/Model/ProductModelSpec.php [hunk 1]
```diff
@@ -193,7 +193,7 @@ function it_gets_the_code_as_label_if_the_label_value_data_is_empty(
         FamilyInterface $family,
         AttributeInterface $attributeAsLabel,
         ValueCollectionInterface $values,
-        ValueInterface $nameValue
+        \Akeneo\Pim\Enrichment\Component\Product\Model\ValueInterface $nameValue
```
Comment (arnolanglade) [inline src/Akeneo/Pim/Enrichment/Component/spec/Product/Model/ProductModelSpec.php]: revert

File: src/Akeneo/Pim/Enrichment/Component/spec/Product/Model/ProductSpec.php [hunk 2]
```diff
@@ -310,7 +310,7 @@ function it_gets_the_identifier_as_label_if_the_label_value_data_is_empty(
         AttributeInterface $attributeAsLabel,
         ValueCollectionInterface $values,
         ValueInterface $nameValue,
-        ValueInterface $identifier
+        \Akeneo\Pim\Enrichment\Component\Product\Model\ValueInterface $identifier
```
Comment (arnolanglade) [inline src/Akeneo/Pim/Enrichment/Component/spec/Product/Model/ProductSpec.php]: revert

File: src/Akeneo/Pim/Enrichment/Component/spec/Product/Model/ProductModelSpec.php (hunk 1 above)
Comment (jjanvier) [reply src/Akeneo/Pim/Enrichment/Component/spec/Product/Model/ProductModelSpec.php]: fixed

File: src/Akeneo/Pim/Enrichment/Component/spec/Product/Model/ProductSpec.php (hunk 2 above)
Comment (jjanvier) [reply src/Akeneo/Pim/Enrichment/Component/spec/Product/Model/ProductSpec.php]: fixed

**Pull request merged**